web/src/main/resources/schema/jboss-web_10_1.xsd

```diff
@@ -0,0 +1,599 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2012, Red Hat, Inc., and individual contributors
```

Comment:
Note that this is the file that people will actually be looking at, wrong year here might give them an impression its honestly old, so its a UX thing to get this right.